application/core/ConsoleApplication.php

```diff
@@ -44,15 +44,27 @@ public function __construct($aApplicationConfig = null)
 
         // Set webroot alias.
         Yii::setPathOfAlias('webroot', realpath(Yii::getPathOfAlias('application').'/../'));
-
         /* Because we have app now : we have to call again the config : can be done before : no real usage of url in console, but usage of getPathOfAlias */
         $coreConfig = require(__DIR__.'/../config/config-defaults.php');
         $consoleConfig = require(__DIR__.'/../config/console.php'); // Only for console : replace some config-defaults
         $emailConfig = require(__DIR__.'/../config/email.php');
         $versionConfig = require(__DIR__.'/../config/version.php');
         $updaterVersionConfig = require(__DIR__.'/../config/updater_version.php');
-        $lsConfig = array_merge($coreConfig, $consoleConfig, $emailConfig, $versionConfig, $updaterVersionConfig);
 
+        $lsConfig = array_merge(
+            $coreConfig,
+            $consoleConfig,
+            $emailConfig,
+            $versionConfig,
+            $updaterVersionConfig
+        );
+
+        if (file_exists(__DIR__.'/../config/security.php')) {
```

Comment:
Should probably use `$configdir` here instead of hard-coded location?

Reply:
Sorry, Denis...

Reply:
Oh, the location is hard-coded everywhere above. Strange.

Reply:
$configdir is broken in some server …

I think config-default `$config['rootdir'] = getcwd();` muts be fixed for console.

A lot of user don't know how to create a multi line cron … on server.

Reply:
> Should probably use `$configdir` here instead of hard-coded location?

This is done after , before this fix : default security are never read … in specific server …

Reply:
Hm OK. :thinking: :cow:

Reply:
https://github.com/LimeSurvey/LimeSurvey/pull/1424#issuecomment-631497967